wait.sh

```diff
@@ -73,7 +80,13 @@ function main() {
     if [[ $loop -lt 1 ]]
     then
       >&2 echo "[ERROR] Timeout waiting for docker-compose to start"
-      >&2 docker-compose logs
+      for s in "${services[@]}"
+      do
+        if ! grep -q "$(docker-compose -p "${PROJECT}" ps -q "$s")" <<< "$(docker ps -q --no-trunc)"
```

Comment:
I just made this the same condition as https://github.com/greenpeace/planet4-docker-compose/pull/68/files#diff-f6a7e0f738cecc51c8a60ce2dad28b8cd8f16472e81c63b75a55e5cc63381dd2R37